### PR TITLE
Fix rubocop issues in step definitions

### LINKED ISSUE #355: Fix rubocop issues in step definitions

### PR DESCRIPTION
**Pull request by arun1595**
Fixes #355

### CODE REVIEW

File: features/step_definitions/confirmation_steps.rb [hunk 1]
```diff
@@ -1,14 +1,17 @@
-Given(/^I have a unconfirmed account that is over (\d+) days old$/) do |arg1|
-  @user = FactoryGirl.create(:user, confirmed_at: nil, created_at: 2.days.ago)
-  @user.update_attributes(confirmation_sent_at: 2.days.ago)
+Given(/^I have a unconfirmed account that is over (\d+) days old$/) do |numdays|
+  @user = FactoryGirl.create(:user,
```
Comment (KevinMulhern) [inline features/step_definitions/confirmation_steps.rb]: I understand rubocop is saying this is a 80 character line violation. Sometimes its better to have a line over 80 characters though if it makes it easier to read. I think this is one of those cases, this code is very hard to read when it's like this.

Comment (KevinMulhern) [inline features/step_definitions/confirmation_steps.rb]: What about doing something like this instead
```
@user = FactoryGirl.create(
                 :user,
                 confirmed_at: nil,
                 created_at: numdays.to_i.days.ago
               )

  @user.update_attributes(confirmation_sent_at: numdays.to_i.days.ago)
```

Comment (arun1595) [inline features/step_definitions/confirmation_steps.rb]: I agree. Can you please check if it is better now?

**Comment by KevinMulhern** [comment]
👍 LGTM

**Pull request merged**